<launch>
  <node pkg="rosbag" type="play" name="rosbag" args="-d 2 /home/denise/bagfiles/dataset1/2015-01-16-18-37-11.bag"/>
  <node name="extract" pkg="image_view" type="extract_images" respawn="false" output="screen" cwd="ROS_HOME" args= "_sec_per_frame:=0.005" >
    <remap from="image" to="/magnet_track/output_video"/>
  </node>
</launch>
